src/Analysis/Ast/Impl/Diagnostics/ErrorCodes.cs

```diff
@@ -25,5 +25,6 @@ public static class ErrorCodes {
         public const string UndefinedVariable = "undefined-variable";
         public const string VariableNotDefinedGlobally= "variable-not-defined-globally";
         public const string VariableNotDefinedNonLocal = "variable-not-defined-nonlocal";
+        public const string TypeVarNoArguments = "type-var-no-arguments";
```

Comment:
A bit too generic since there may be diagnostics on constraints (TypeVar second argument). Probably `typevar` as a single word. `typevar-missing-first-argument` will do.

Reply:
Ok I can change the name.